wrappers/opencv/face/CMakeLists.txt

```diff
@@ -0,0 +1,20 @@
+#  minimum required cmake version: 3.1.0
+cmake_minimum_required(VERSION 3.1.0)
+
+project(RealSenseOpencvFaceExample)
+
+add_executable(rs-face-cv rs-face-cv.cpp)
+set_property(TARGET rs-face-cv PROPERTY CXX_STANDARD 11)
+target_link_libraries(rs-face-cv ${DEPENDENCIES})
+set_target_properties (rs-face-cv PROPERTIES
+	FOLDER "Examples/OpenCV"
+)
+
+install(
+	TARGETS
+
```

Comment:
Extra spaces

Reply:
This file needs to be taken out of the changes -- the opencv/face/ example is not going in...